### ISSUE TITLE
Does this work with the latest 1.0 NR release?

### ISSUE THREAD

**Issue by chaeron** [btsimonh/node-red-contrib-flowman#6]
Does this work with the latest 1.0 NR release?

**Comment by btsimonh** [comment]
yes;  I've been on 0.17 for ages in production, but moving to 1.0.1+ now.  No issues with this bit so far, which was a nice surprise :).

**Comment by chaeron** [comment]
Cool....I'll give it a try!  Thanks.

Though I think this functionality should be a core part of NR....or at least a standard/supported node. ;-)

**Comment by chaeron** [comment]
Dumb question....when you add a flow using flowman, will it deploy it, or do you still have to do that through the editor?

**Comment by btsimonh** [comment]
"Though I think this functionality should be a core part of NR" - Nick's not keen, he believes everything is possible through the published http api; but in my case admin is disabled.

Yes, it's deployed.  I believe 'enabled' is honored, so if you store a flow disabled, and want it enabled after add, then modify the disabled flag.
If you have configuration nodes, be aware they will be stored, and then made so that they are owned by the flow.  Subflows are also collapsed into the flow (not visually pretty - I never use subflows anyway).

Upon deployment, if you have the admin interface open, it recognises that something changed, and I normally just F5 refresh.

Home use or commercial?  If commercial, I have a whole framework for deploying flows wherever they are needed, and geared up for doing a pay-per-use cost split with the flow author - basically, my business is built on this node!
